### PR TITLE
Added type definitions for website-scraper v1.2.x

### PR DESCRIPTION
**Pull request by screendriver**

### CODE REVIEW

**Comment by vvakame** [comment]
oh, sorry. could you change `website-scraper/index-tests.ts` to `website-scraper/website-scraper-tests.ts` again?

**Comment by vvakame** [comment]
*website-scraper/index.d.ts*

Checklist

* [X] is correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/website-scraper - https://github.com/s0ph1e/node-website-scraper
* [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (website-scraper-tests.ts or website-scraper-tests.tsx)
* [ ] pass the Travis CI test?

File: website-scraper/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```
Comment (screendriver) [inline website-scraper/tsconfig.json]: Done

File: website-scraper/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// Type definitions for website-scraper v1.2.x
+// Project: https://github.com/s0ph1e/node-website-scraper
+// Definitions by: Christian Rackerseder <https://www.echooff.de>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "website-scraper" {
```
Comment (screendriver) [inline website-scraper/index.d.ts]: Hmm what exactly do you mean? I don't quite understand your question 🤔 
Do you mean that I don't have to write a `declare module` and should export the function directly?

**Comment by screendriver** [comment]
I renamed the test file ;-)

File: website-scraper/index.d.ts (hunk 2 above)
Comment (vvakame) [inline website-scraper/index.d.ts]: yes. like this https://github.com/DefinitelyTyped/DefinitelyTyped/pull/12038/commits/e52e881a914360fd5f06245adb7522931d336e80?w=

**Comment by screendriver** [comment]
I replaced module with namespace declaration

File: website-scraper/index.d.ts (hunk 2 above)
Comment (screendriver) [inline website-scraper/index.d.ts]: Done

**Pull request merged**